src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -180,6 +182,15 @@ private void initialiseRuntime(final String resourceType,
         }
     }
 
+    private static String convertStackTraceToString(Throwable throwable) {
```

Comment:
Can we use `ExceptionUtils` from apache common lang?  it has methods to convert as string.